WaterMinder/Handbook/Cell/HandbookChallengesViewCell.swift

```diff
@@ -0,0 +1,141 @@
+        
+        
+        contentView.addSubview(leftWaterImageView)
+        leftWaterImageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        contentView.addSubview(rightChallengeLabel)
+        
+        contentView.addSubview(rightNameChallengeLabel)
+        rightNameChallengeLabel.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
+            leftWaterImageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 20),
+            leftWaterImageView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 10),
+            leftWaterImageView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            leftWaterImageView.widthAnchor.constraint(equalToConstant: 110),
+            
+            rightChallengeLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 30),
+            rightChallengeLabel.leadingAnchor.constraint(equalTo: leftWaterImageView.trailingAnchor, constant: 30),
+            
+            rightNameChallengeLabel.topAnchor.constraint(equalTo: rightChallengeLabel.bottomAnchor, constant: 10),
+            rightNameChallengeLabel.leadingAnchor.constraint(equalTo: leftWaterImageView.trailingAnchor, constant: 30),
+            rightNameChallengeLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -10),
+            
+            
+            
+            rightWaterImageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 20),
+            rightWaterImageView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -10),
+            rightWaterImageView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            rightWaterImageView.widthAnchor.constraint(equalToConstant: 110),
+
+            leftChallengeLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 30),
+            leftChallengeLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 30),
+            
+            leftNameChallengeLable.topAnchor.constraint(equalTo: leftChallengeLabel.bottomAnchor, constant: 10),
+            leftNameChallengeLable.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 30),
+            leftNameChallengeLable.trailingAnchor.constraint(equalTo: rightWaterImageView.leadingAnchor, constant: -10)
+        ])
+    }
+    
+    func set(_ itemsInWater: ChallengesViewModel) {
+        
+        leftWaterImageView.image = itemsInWater.leftImage
+
+        rightWaterImageView.image = itemsInWater.rightImage
+        
+        contentView.backgroundColor = itemsInWater.backgroundColor
+    
+        leftChallengeLabel.text = itemsInWater.leftChallengeText
+        rightChallengeLabel.text = itemsInWater.rightChallengeText
+        
+        leftNameChallengeLable.text = itemsInWater.leftChallengeNameText
+        leftNameChallengeLable.textColor = .white
```

Comment:
the same as labels above, u have no dependency on view model here
set up params when u create object